localstack/services/kms/packages.py

```diff
@@ -0,0 +1,36 @@
+import os
+import platform
+from typing import List
+
+from localstack.constants import KMS_URL_PATTERN
+from localstack.packages import Package, PackageInstaller
+from localstack.packages.core import PermissionDownloadInstaller
+from localstack.utils.platform import get_arch
+
+
+class KMSLocalPackage(Package):
+    def __init__(self):
+        super().__init__("LocalKMS", "latest")
+
+    def get_versions(self) -> List[str]:
+        return ["latest"]
+
+    def _get_installer(self, version: str) -> PackageInstaller:
+        return KMSLocalPackageInstaller("local-kms", version)
+
+
+class KMSLocalPackageInstaller(PermissionDownloadInstaller):
+    @staticmethod
+    def _get_local_arch():
+        return f"{platform.system().lower()}-{get_arch()}"
```

Comment:
I think it's not necessary to extract this to a new function. The `_get_download_url` function is responsible of creating the download URL.

Reply:
Thanks for pointing it out, this one was not using the DownloadInstaller features properly. Fixed and reworked :+1: